index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Emmanuel Hagan" />
+  <title>Hagan's Portifio</title>
+    <!-- Stylesheet -->
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+  <!--  Header  -->
+  <header class="toolbar-container">
+      <a href="#" class="logo-heading" > Hagan</a>
+   <!---  
+      <nav class="hamburger-menu">
+       <ul class="header-nav-links">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li> 
+        </ul>
+      </nav>
+    -->
+      <img class="icon" src="assets/image/icon-menu.png"  alt="Icon menu">
```

Comment:
- [ ] please note that this `img` tag should be wrapped inside a `nav` tag.

Reply:
- Remove all comments on the code.